spec/controllers/openid_connect/authorization_controller_spec.rb

```diff
@@ -131,6 +132,7 @@
               with(
                 'SP redirect initiated',
                 ial: 1,
+                sign_in_duration_seconds: 120,
```

Comment:
I'm worried about clock skew contributing to flakey specs here, we probably want to freeze time with a specific time and specifically specify 120 seconds ago for the time

Reply:
Would that mean wrapping the expect() blocks like this?

diff --git a/spec/controllers/openid_connect/authorization_controller_spec.rb b/spec/controllers/openid_connect/authorization_controller_spec.rb
index 7b1ca74e73..fdc40c393d 100644
--- a/spec/controllers/openid_connect/authorization_controller_spec.rb
+++ b/spec/controllers/openid_connect/authorization_controller_spec.rb
@@ -142,15 +142,17 @@ RSpec.describe OpenidConnect::AuthorizationController, allowed_extra_analytics:
               user_sp_authorized: true,
               code_digest: kind_of(String),
             )
-            expect(@analytics).to have_logged_event(
-              'SP redirect initiated',
-              ial: 1,
-              billed_ial: 1,
-              sign_in_duration_seconds: 60,
-              sign_in_flow:,
-              acr_values: 'http://idmanagement.gov/ns/assurance/ial/1',
-              vtr: nil,
-            )
+            freeze_time do
+              expect(@analytics).to have_logged_event(
+                'SP redirect initiated',
+                ial: 1,
+                billed_ial: 1,
+                sign_in_duration_seconds: 60,
+                sign_in_flow:,
+                acr_values: 'http://idmanagement.gov/ns/assurance/ial/1',
+                vtr: nil,
+              )


Unrelated but I arbitrarily changed to 60 seconds trying something different.
`session[:sign_in_page_visited_at] = 1.minute.ago.to_s`

Reply:
`freeze_time` usually goes in an `around` or a `before` block